### PR TITLE
[minor] Make install plan configurable for all subscriptions in gitops

### PR DESCRIPTION
**Pull request by sanjayprab**
Making installplan approval configurable for all subscription in gitops

https://jsw.ibm.com/browse/MASCORE-3901

### CODE REVIEW

File: image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-cp4d-services-base.yaml.j2 [hunk 1]
```diff
@@ -2,7 +2,12 @@ merge-key: "{{ ACCOUNT_ID }}/{{ CLUSTER_ID }}/{{ MAS_INSTANCE_ID }}"
...
   ccs_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_CCS_CHANNEL }}>"
+  cpd_ccs_install_plan : "{{ CPD_CCS_INSTALL_PLAN }}"
   datarefinery_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_DATAREFINERY_CHANNEL }}>"
+  cpd_datarefinery_install_plan : "{{ CPD_DATAREFINERY_INSTALL_PLAN }}"
   ws_runtimes_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_WS_RUNTIMES_CHANNEL }}>"
+  cpd_ws_install_plan : "{{ CPD_WS_INSTALL_PLAN }}"
   opencontent_rabbitmq_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_OPENCONTENT_RABBITMQ_CHANNEL }}>"
+  rabbitmq_install_plan : "{{ RABBITMQ_INSTALL_PLAN }}"
   opencontent_elasticsearch_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_OPENCONTENT_ELASTICSEARCH_CHANNEL }}>"
+  elasticsearch_install_plan: "{{ ELASTICSEARCH_INSTALL_PLAN }}"
```
Comment (whitfiea) [inline image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-cp4d-services-base.yaml.j2:13]: Nothing is setting these variables in this template. ~~I think we should remove these are these should be controlled by cp4d~~

Edit: After looking at the gitops changes I can see we set these subscriptions, so we need to add these to the functions as well

File: image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-spss.yaml.j2 [hunk 2]
```diff
@@ -7,4 +7,6 @@ ibm_spss:
   cpd_service_storage_class: "{{ CPD_SERVICE_STORAGE_CLASS }}"
   spss_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_SPSS_CHANNEL }}>"
   spss_version: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_SPSS_VERSION }}>"
+  spss_install_plan: "{{ SPSS_INSTALL_PLAN }}"
   canvasbase_channel: "<path:{{ SECRETS_PATH }}:{{ SECRET_KEY_CANVASBASE_CHANNEL }}>"
+  canvasbase_install_plan: "{{ CANVASBASE_INSTALL_PLAN }}"
```
Comment (whitfiea) [inline image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-spss.yaml.j2:12]: canvasbase_install_plan is not set by anything.

**Review by whitfiea — CHANGES_REQUESTED** [review]

**Comment by whitfiea** [comment]
Also need to update title and description

File: image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-cp4d-services-base.yaml.j2 (hunk 1 above)
Comment (sanjayprab) [reply image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-cp4d-services-base.yaml.j2:13]: Done

File: image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-spss.yaml.j2 (hunk 2 above)
Comment (sanjayprab) [reply image/cli/mascli/templates/gitops/appset-configs/cluster/instance/ibm-spss.yaml.j2:12]: Done

File: tekton/src/tasks/dependencies/cp4d-service-update.yml.j2 [hunk 3]
```diff
@@ -79,6 +79,6 @@ spec:
       command:
         - /opt/app-root/src/run-role.sh
         - cp4d_service
-      image: quay.io/ibmmas/cli:latest
+      image: quay.io/ibmmas/cli:11.3.0-pre.mascore3901
```
Comment (whitfiea) [inline tekton/src/tasks/dependencies/cp4d-service-update.yml.j2]: 11.3.0-pre.mascore3901 needs to be removed

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: tekton/src/tasks/dependencies/cp4d-service-update.yml.j2 (hunk 3 above)
Comment (sanjayprab) [reply tekton/src/tasks/dependencies/cp4d-service-update.yml.j2]: done

**Review by whitfiea — APPROVED** [review]

**Pull request merged**